lte/gateway/c/session_manager/SessionProxyResponderHandler.cpp

```diff
@@ -105,15 +104,35 @@ void SessionProxyResponderHandlerImpl::PolicyReAuth(
   });
 }
 
-SessionMap SessionProxyResponderHandlerImpl::get_sessions_for_charging(
-    const ChargingReAuthRequest& request) {
-  SessionRead req = {request.sid()};
-  return session_store_.read_sessions(req);
-}
-
-SessionMap SessionProxyResponderHandlerImpl::get_sessions_for_policy(
-    const PolicyReAuthRequest& request) {
-  SessionRead req = {request.imsi()};
-  return session_store_.read_sessions(req);
+void SessionProxyResponderHandlerImpl::AbortSession(
+    ServerContext* context, const AbortSessionRequest* request,
+    std::function<void(Status, AbortSessionResult)> response_callback) {
+  PrintGrpcMessage(static_cast<const google::protobuf::Message&>(*request));
+  const auto imsi       = request->user_name();
+  const auto session_id = request->session_id();
+  MLOG(MINFO) << "Received an ASR for session_id " << session_id;
+  enforcer_->get_event_base().runInEventBaseThread([this, imsi, session_id,
+                                                       response_callback]() {
+    auto session_map = session_store_.read_sessions({imsi});
+    SessionUpdate update =
```

Comment:
what about using 
```
SessionUpdate session_uc 
```

During this time I have been modifying the counting code i realized there is lots of `updates` (criterias and network updates).  Lets use the `session_uc` `credit_uc` ... for any update criteria

Reply:
Yeah good call will change.

Reply:
Ok, so I think `SessionUpdate` -> `session_updates` is ok.
`SessionStateUpdateCriteria` -> `session_uc`
`SessionCreditUpdateCriteria` -> `credit_uc`
how does that sound?

Reply:
isn't `SessionUpdate` an update criteria?

Reply:
it is a map of update criterias, so I think calling it 'updates' is reasonable enough. But I agree about `session_uc` and `credit_uc`, it's good to distinguish them with fixed names. I always get confused which is which now ;)